wasm/Makefile

```diff
@@ -148,7 +148,6 @@ $(WASM_OBJ_DIR)/opa.wasm: $(OBJS) $(CPP_OBJS) $(LIB_OBJS) $(LIB_MPDEC_OBJS) $(LI
 			--allow-undefined-file=src/undefined.symbols \
 			--import-memory \
 			--no-entry \
-			--export-all \
```

Comment:
Should we make the same change to the test module below? If not, it might be useful to include a note (either in the commit message or the makefile) about why they're treated differently (debugging?)

Reply:
Good point! There's no reason not to do it. Let's see how it goes 😄